CHANGELOG-PENDING.md

```diff
@@ -10,11 +10,11 @@ Month, DD, YYYY
 
 ### FEATURES
 
-- [go package] (Link to PR) Description @username
+- [indexer] [Implement block and transaction indexing, enable TxSearch RPC endpoint #202](https://github.com/celestiaorg/optimint/pull/202) [@mattdf](https://github.com/mattdf)
```

Comment:
This is unrelated but was just added while doing the changelog anyways?

Reply:
I just moved this from "Improvements" to "Features", while updating changelog